src/ogma/features/nodeGrouping.ts

```diff
@@ -282,4 +331,71 @@ export class NodeGroupingTransformation {
       )}-${propertyValue}`
     );
   }
+
+  public _runCircularLayout({
```

Comment:
this is duplicate of circularLayout in ogmaTool,

Reply:
@aminerhanemi I didn't know where is the correct place to put it. What's the best way?

Reply:
left it in the NodeGrouping file